platforms/web/cypress/e2e/clipboard/cut.spec.ts

```diff
@@ -20,7 +20,8 @@ describe('Cut', () => {
     const editor = '.editor:not([disabled])[contenteditable="true"]';
 
     it('remove text that is cut to clipboard', { browser: 'electron' }, () => {
-        cy.visit('/').wait(300);
+        cy.visit('/');
+        cy.get(editor).wait(500);
```

Comment:
I wonder if we can wait until the `contentEditable` field is at true ?

```ts
cy.get('#editor',  { timeout: 1000 }).should('have.attr', 'contentEditable', true)
```

(I'm not familiar with the cypress syntax, so it should not work out of box ^^)

Reply:
The `editor` variable already says that `[contenteditable="true"]` so it should not match the `get` until it is editable.

I don't know why some of the characters don't type :-(

Reply:
Unless my CSS selector is wrong?

Reply:
The selectors seems to be good

Reply:
Merging, in case it helps :-)

Reply:
I did some local testing and it seems slightly more reliable so fingers crossed.